apps/src/gamelab/GameLabP5.js

```diff
@@ -435,6 +646,7 @@ GameLabP5.prototype.getCustomMarshalBlockedProperties = function () {
 
 GameLabP5.prototype.getCustomMarshalObjectList = function () {
   return [
+    { instance: GameLabGame },
```

Comment:
Maybe. The tricky thing about `p5play` is that they don't typically use `prototype` at all. It would be easier to extend if they did. And probably more efficient since we wouldn't have to patch each object as it is instantiated.